src/workerd/server/tests/python/worker-entrypoint/worker.py

```diff
@@ -11,6 +13,8 @@ def __init__(self, state, env):
         self.counter = 0
 
     async def rpc_trigger_func(self):
+        # Test that entropy works in WorkerEntrypoint...
+        os.urandom(4)
```

Comment:
This doesn't actually reproduce the problem but it doesn't hurt to have it here.
The internal test actually do.